sound/soc/sof/amd/Kconfig

```diff
@@ -31,4 +31,14 @@ config SND_SOC_SOF_AMD_RENOIR
 	select SND_SOC_SOF_AMD_COMMON
 	help
 	  Select this option for SOF support on AMD Renoir platform
+
```

Comment:
what platform driver are you talking about in the commit message?

Reply:
its rembrandt only